src/server/api/forms/courses.js

```diff
@@ -0,0 +1,48 @@
+const express = require('express');
+
+const router = express.Router();
+const registrar = require('../../services/registrar');
+
+router.post('/', (req, res, err) => {
+  (async () => {
+    const { termCode } = req.body;
+    const { subjectAreaCode: department } = req.body;
```

Comment:
@tatikamsetti So this was passed in from the frontend as subjectAreaCode but then you renamed it to department, but then use it as subjectAreaCode against below? Why not just use this as subjectAreaCode?

const { subjectAreaCode } = req.body;

Then you can combine it with the line above:

const { termCode, subjectAreaCode } = req.body;